app/src/main/java/com/babylon/wallet/android/presentation/settings/securitycenter/SecurityCenterScreen.kt

```diff
@@ -352,14 +352,26 @@ private fun BackupConfigurationCard(needsAction: Boolean, onBackupConfigurationC
 @Composable
 fun SecurityProblem.toProblemHeading(): String {
     return when (this) {
-        is SecurityProblem.EntitiesNotRecoverable -> stringResource(
-            id = R.string.securityCenter_problem3_heading,
-            accountsNeedBackup,
-            personasNeedBackup
-        )
+        is SecurityProblem.EntitiesNotRecoverable -> {
+            val accountsString = if (accountsNeedBackup == 1) {
```

Comment:
It would have been nice to use plurals here (not sure if it's possible with the current localization repo setup)

Reply:
not possible, we use plurals by having separate string defined for 1 and n variant for each personas and accounts, and eventually we combine those 2 separate counts into one string